driver/ppm_fillers.c

```diff
@@ -934,19 +939,17 @@ bool ppm_is_upper_layer(struct file *exe_file){
 	if(sb)
 	{
 		sb_magic = sb->s_magic;
-		struct ovl_entry *oe = (struct ovl_entry*)(exe_file->f_path.dentry->d_fsdata);
+		oe = (struct ovl_entry*)(exe_file->f_path.dentry->d_fsdata);
```

Comment:
Probably here we only need to switch the order, so
```
struct ovl_entry *oe = (struct ovl_entry*)(exe_file->f_path.dentry->d_fsdata);
sb_magic = sb->s_magic;
```

instead of:

```
sb_magic = sb->s_magic;
struct ovl_entry *oe = (struct ovl_entry*)(exe_file->f_path.dentry->d_fsdata);
```

Reply:
Just updated! ty! 😄